l10n_br_hr/model/hr_employee.py

```diff
@@ -182,3 +183,17 @@ def _validate_cpf(self):
         ('f', 'Female')])
     dependent_rg = fields.Char(string='RG')
     dependent_cpf = fields.Char(string='CPF')
+
+    have_alimony = fields.Boolean(string='Have a alimony')
+
+    partner_id = fields.Many2one(
```

Comment:
Quando for processar pagamentos, é preciso de um partner para ser beneficiario do pagamento. No caso de pensão, precisamos apontar o beneficiário (res_partner) e suas informações bancárias para efetuar pagamentos diretamente ao dependente do funcionário.